src/translations/en/languages.json

```diff
@@ -1,18 +1,18 @@
 {
-  "da": "Danish",
-  "en": "English",
-  "fi": "Finnish",
-  "fr": "French",
-  "is": "Icelandic",
-  "it": "Italian",
-  "nl": "Dutch",
-  "nb": "Norwegian, Bokmål",
-  "nn": "Norwegian, Nynorsk",
-  "pt": "Portuguese",
-  "ru": "Russian",
-  "se": "Sami",
-  "es": "Spanish",
-  "sv": "Swedish",
-  "de": "German",
-  "xx": "Other"
+  "dan": "Danish",
+  "eng": "English",
+  "fin": "Finnish",
+  "fre": "French",
+  "ice": "Icelandic",
+  "ita": "Italian",
+  "dut": "Dutch",
+  "nob": "Norwegian, Bokmål",
+  "nno": "Norwegian, Nynorsk",
+  "por": "Portuguese",
+  "rus": "Russian",
+  "smi": "Sami",
+  "spa": "Spanish",
+  "swe": "Swedish",
+  "ger": "German",
+  "mis": "Other"
 }
```

Comment:
hvorfor er det 3 bokstaver nå?

Reply:
egentlig bare for å matche verdier som kreves av backend bedre, som er type `http://www.lexvo.org/page/iso639-3/nob`.
Ingenting i veien for å fortsette med to bokstaver, hvis du syns det er ryddigere?

Reply:
jaok, neida, så lenge det er en grunn til det så er det nice 👌